players/utils.py

```diff
@@ -14,7 +21,7 @@ def add_player_to_team_and_group(team, player):
     team.pending_player.clear()
 
 
-def last_and_first_picking_persons(team):
+def last_and_first_picking_persons(team: Team) -> (Profile, Profile):
```

Comment:
```suggestion
def last_and_first_picking_persons(team: Team) -> tuple[Profile, Profile]:
```

lub w przypadku pythona niższego niż 3.9:
```suggestion
from typing import Tuple


def last_and_first_picking_persons(team: Team) -> Tuple[Profile, Profile]:
```